service/network/responses.go

```diff
@@ -0,0 +1,46 @@
+package network
```

Comment:
The idea of handling the errors on a common way is cool, but this functions could go within the controller layer with no problem. The controller is the layer that should handle communication with the client after all.

Reply:
*